builtin/providers/aws/resource_aws_elb_load_balancer_policy.go

```diff
@@ -0,0 +1,350 @@
+package aws
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/awserr"
+	"github.com/aws/aws-sdk-go/service/elb"
+	"github.com/hashicorp/terraform/helper/schema"
+)
+
+func resourceAwsLoadBalancerPolicy() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceAwsLoadBalancerPolicyCreate,
+		Read:   resourceAwsLoadBalancerPolicyRead,
+		Update: resourceAwsLoadBalancerPolicyUpdate,
+		Delete: resourceAwsLoadBalancerPolicyDelete,
+
+		Schema: map[string]*schema.Schema{
+			"load_balancer_name": &schema.Schema{
+				Type:     schema.TypeString,
+				Required: true,
+				ForceNew: true,
+			},
+
+			"policy_name": &schema.Schema{
+				Type:     schema.TypeString,
+				Required: true,
+				ForceNew: true,
+			},
+
+			"policy_type_name": &schema.Schema{
+				Type:     schema.TypeString,
+				Required: true,
+				ForceNew: true,
+			},
+
+			"policy_attribute": &schema.Schema{
+				Type:     schema.TypeSet,
+				Optional: true,
+				Elem: &schema.Resource{
+					Schema: map[string]*schema.Schema{
+						"name": &schema.Schema{
+							Type:     schema.TypeString,
+							Optional: true,
+						},
+
+						"value": &schema.Schema{
+							Type:     schema.TypeString,
+							Optional: true,
+						},
+					},
+				},
+			},
+		},
+	}
+}
+
+func resourceAwsLoadBalancerPolicyCreate(d *schema.ResourceData, meta interface{}) error {
+	elbconn := meta.(*AWSClient).elbconn
+
+	attributes := []*elb.PolicyAttribute{}
+	if attributedata, ok := d.GetOk("policy_attribute"); ok {
+		attributeSet := attributedata.(*schema.Set).List()
+		for _, attribute := range attributeSet {
+			data := attribute.(map[string]interface{})
+			attributes = append(attributes, &elb.PolicyAttribute{
+				AttributeName:  aws.String(data["name"].(string)),
+				AttributeValue: aws.String(data["value"].(string)),
+			})
+		}
+	}
+
+	lbspOpts := &elb.CreateLoadBalancerPolicyInput{
+		LoadBalancerName: aws.String(d.Get("load_balancer_name").(string)),
+		PolicyName:       aws.String(d.Get("policy_name").(string)),
+		PolicyTypeName:   aws.String(d.Get("policy_type_name").(string)),
+		PolicyAttributes: attributes,
```

Comment:
what happens if PolicyAttributes is empty? Should we perform the same check as expressed before?